pages/1.8/release-notes/index.md

```diff
@@ -2,8 +2,8 @@
 layout: layout.pug
```

Comment:
It looks like you might have moved this file from its original location? If so it will need a redirect.